ocsns/sns.go

```diff
@@ -12,7 +12,7 @@ import (
 	"go.opencensus.io/trace"
 )
 
-// An Option function customises a clients configuration
+// An Option function customizes a clients configuration
```

Comment:
I'm British so this hurts 😛

Reply:
Lol.. So it is better the previous ya @krak3n ?

Reply:
nah, I have come to accept American English is the most common form in programming so it's cool :+1: